filebeat/module/apache2/module.yml

```diff
@@ -1 +0,0 @@
-movedTo: apache
```

Comment:
This was added to don't break existing configurations.

Reply:
this was removed accidentially during the rebase. Will readd it. It should also bring the tests back to green in autodiscovery.